packages/@atjson/source-ckeditor/test/source-classic-build-editor-test.ts

```diff
@@ -0,0 +1,280 @@
+import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
```

Comment:
This is a copy of the old tests which test fromRaw() from the Editor.model.document. There are new tests which test fromRaw() using a DocumentFragment below.